timesketch/frontend-v3/package.json

```diff
@@ -14,7 +14,8 @@
     "roboto-fontface": "*",
     "vue": "^3.4.31",
     "vuetify": "^3.6.11",
-    "axios": "^1.7.7"
+    "axios": "^1.7.7",
+    "tiny-emitter": "^2.1.0"
```

Comment:
I use "mitt" in openrelik, another emitter library. What is the benefit of tiny-emitter? Or is it just another implementation of the same thing? :)

Reply:
Ah I see in the Vue migration guide. 
"This provides the same event emitter API as in Vue 2."

Nice, then we don't have to change the caller code.